lrpar/cttests/build.rs

```diff
@@ -13,5 +15,23 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     for entry in glob("src/*.test")? {
         run_test_path(entry.unwrap())?;
     }
+
+    cfg_aliases! {
+        // Platforms
+        wasm32_unknown: { all(target_arch = "wasm32", target_os="unknown", target_vendor="unknown") },
+    }
+
+    CTLexerBuilder::new()
+        .rust_edition(lrlex::RustEdition::Rust2021)
```

Comment:
One last thought is that I don't see any reason we can't move this build.rs, calc_wasm.l, and calc_wasm.y into a `calc_wasm.test` file.